examples/bash_automatic_cd.sh

```diff
@@ -7,14 +7,14 @@
 # To undo the effect of this function, you can type "cd -" to return to the
 # original directory.
 
-function ranger-cd {
-    tempfile="$(mktemp -t tmp.XXXXXX)"
-    ranger --choosedir="$tempfile" "${@:-$(pwd)}"
-    test -f "$tempfile" &&
-    if [ "$(cat -- "$tempfile")" != "$(echo -n `pwd`)" ]; then
-        cd -- "$(cat "$tempfile")"
+ranger-cd() {
+    local temp_file chosen_dir
+    temp_file="$(mktemp)"
```

Comment:
I'd rather change the pattern to `mktemp -t ranger-cd.XXXXXX` or something. I've had platforms like darwin complain when the X's aren't part of the pattern when called from a script even though it works fine on the command line.

Reply:
done